src/lib.rs

```diff
@@ -379,5 +382,6 @@ pub fn new_vm() -> RootedThread {
     ::vm::channel::load(&vm).expect("Loaded channel library");
     ::vm::debug::load(&vm).expect("Loaded debug library");
     ::io::load(&vm).expect("Loaded IO library");
+    ::regex_bind::load(&vm).expect("Loaded regex library");
```

Comment:
I think its fine to always import it for now as the library is so small anyway. Its easy enough to make this an optional import at a later point anyway